tasks/db_deploy/test/manual_tests/README.md

```diff
@@ -70,126 +69,15 @@ Once all tests are complete perform the following:
-
-```bash
-{"message": "Beginning manual test.", "timestamp": "2021-04-28T22:19:52.948107", "level": "info"}
-{"message": "Creating root user connection object.", "timestamp": "2021-04-28T22:19:52.948701", "level": "debug"}
-{"message": "Database set to postgres for the connection.", "timestamp": "2021-04-28T22:19:52.949417", "level": "debug"}
-{"message": "Creating URL object to connect to the database.", "timestamp": "2021-04-28T22:19:52.950417", "level": "debug"}
-{"message": "The ORCA database disaster_recovery does not exist.", "timestamp": "2021-04-28T22:19:53.243872", "level": "critical"}
-Traceback (most recent call last):
-  File "manual_test.py", line 49, in <module>
-      task(get_configuration())
-        File "/data/db_deploy.py", line 60, in task
-            raise Exception("Missing application database.")
-            Exception: Missing application database.
-```
-
-### DNE Test Validation
-
-Verify the following message line and exception appear in the output. Note that
-the time stamp value will be different.
-
-```bash
-{"message": "The ORCA database disaster_recovery does not exist.", "timestamp": "2021-04-28T22:19:53.243872", "level": "critical"}
-Traceback (most recent call last):
-  File "manual_test.py", line 49, in <module>
-      task(get_configuration())
-        File "/data/db_deploy.py", line 60, in task
-            raise Exception("Missing application database.")
-            Exception: Missing application database.
-```
-
-Verify that only the default users, groups and schemas are present in the postgres
-database.
-
-```bash
-postgres=# \dn
-
-  List of schemas
-  Name  |  Owner
---------+----------
- public | postgres
-(1 row)
-
-postgres=# \dg
-                                  List of roles
- Role name |                         Attributes                         | Member of
------------+------------------------------------------------------------+-----------
- postgres  | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
-
-postgres=# \du
-                                  List of roles
- Role name |                         Attributes                         | Member of
------------+------------------------------------------------------------+-----------
- postgres  | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
-
-```
-
-
-### DNE Test Cleanup
-
-No Cleanup is needed from this test.
-
-
 ## Database Fresh Install Test
 
 This test validates that the db_deploy scripts will perform a fresh install if
-the disaster_recovery database exists and no ORCA schemas (dr or orca namespaces)
-exist in the database.
+the server exists without the disaster_recovery database.
 
 ### Database Setup Fresh Install Test
 
 1. Verify that the *disaster_recovery* database does not exist. If it does, remove
    it using the `\i sql/cleanup.sql` command as described in the [DNE Test](#database-setup-dne-test).
-2. Create the *disaster_recovery* database by using the `\i sql/create_database.sql`
```

Comment:
May still want to do this to check logic

Reply:
Hopefully the added output detail is enough

Reply:
Per our conversation add secondary fresh install. test where the disaster database exists but is empty.

Reply:
Added